assets/prototype/application/utilities/money/formatAmount.ts

```diff
@@ -0,0 +1,16 @@
+import parsedAmount from './parsedAmount';
+
+/**
+ * returns a function that when supplied a value for the number of decimal places used by a currency,
+ * returns a second function that can be passed an amount which will then be appropriately formatted
+ *
+ * @param {number} decimalPlaces
+ * @return {Function}
+ */
+const formatAmount = (decimalPlaces: number): ((amount: number | string) => string) => (
```

Comment:
Won't this conflict with https://github.com/eventespresso/event-espresso-core/pull/2177/files#diff-467e0ae393141ac09133b63937c62aaeR12 
?

Reply:
that other PR will supercede this one. I had to base these PRs off of the previous ones to avoid errors